cockpit-*: add page (#9145)
tldr-pages/tldr: pages/linux/cockpit-bridge.md

# cockpit-bridge

> Used to relay messages and commands between the frontend and server in the cockpit suite.
> More information: <https://cockpit-project.org/guide/latest/cockpit-bridge.1.html>.

> - List all cockpit packages:

Reviewer: ```suggestion
- List all cockpit packages:

```

`cockpit-bridge --packages`

> - Show command help:

Reviewer: ```suggestion
- Display help:

```

# cockpit-bridge

> > Used to relay messages and commands between the frontend and server in the cockpit suite.

Reviewer: ```suggestion
> Relay messages and commands between the front end and server in the cockpit suite.
```
